content/class/User.md

```diff
@@ -0,0 +1,134 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/User
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public interface User
+```
+
+`User` is the generic class storing a user who does something on a platform.
+
+| API    | Twitter 1 | Twitter 2 | Mastodon |
+| ------ | --------- | --------- | -------- |
+| Object | [User](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/user) | [User](https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/user) | [Account](https://docs.joinmastodon.org/entities/account/) |
+
+## Properties
+
+### `id` {#property_id}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public string id { get; }
+```
+
+The unique identifier of a user.
+
+| API   | Twitter 1 | Twitter 2 | Mastodon |
+| ----- | --------- | --------- | -------- |
+| Field | `id_str`  | `id`      | `id`     |
+| Type  | string    | string    | string   |
+
+### `name` {#property_name}
```

Comment:
Can we call this "display_name"? Because I think of name (fixed ASCII handle) and display name (pretty name that people shove emojis in) and it's consistent with your comment.